src/index.js

```diff
@@ -19,19 +19,12 @@ const debug = (function () {
   }
 })()
 
-const createSwarm = (hub) => {
-  return swarm(hub, {
-    wrtc: !swarm.WEBRTC_SUPPORT ? require('wrtc') : null,
-    config: config.SWARM_CONFIG
-  })
-}
-
 class Masq {
   /**
    * constructor
    * @param {string} appName - The application name
    */
-  constructor (appName, appDescription, appImageURL, signalhubUrl, baseMasqAppUrl) {
+  constructor (appName, appDescription, appImageURL, _options = {}) {
```

Comment:
you should not prefix wiht an underscore

Reply:
changed